### PR TITLE
[Fix] UNO 487 - Different image size shared stimulus item authoring

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to this PR**

https://oat-sa.atlassian.net/browse/UNO-487

**Description**

- Have an item with a share stimulus, with images that have less than 100%
- When in authoring the images are all showing ass 100%
- In preview the images are showing as they should. Less than 100%. 

I thing the correct view of images should be shown in authoring also.

Pictures added in the shared stimulus are expanded to the size of interaction block, when shared stimulus is added to the interaction, while Authoring an item.

**Steps to reproduce**

- Add shared stimulus with pictures to the interaction in the item.

**Actual result**

Pictures are expanded to the size of interaction block in Authoring mode.

**Expected result**

Pictures in shared stimulus added to the interaction are of the same scale in both Authoring and Preview modes

### CODE REVIEW

**Comment by habemuscode** [comment]
I test that this change don't affect to other images, only `sharedStimulus`

imagen

**Review by bziondik — CHANGES_REQUESTED** [review]
Style issue with an inserted image (not in the shared stimulus)
image
image
Insert image in description, set for example 25%, click outside and it will reduce

**Comment by habemuscode** [comment]
@bziondik  Better approach pushed

imagen

I push manifest version too

**Review by jsconan — CHANGES_REQUESTED** [review]
The code is straight forward and it works in most situations.
Unfortunately, it does not work under IE. See why in my comment below.

File: views/scss/item-creator.scss [hunk 1]
```diff
@@ -794,6 +794,12 @@ img.dragging.ui-draggable-dragging {
                     min-width: 100%;
                 }
 
+                .qti-include {
+                    img {
+                        min-width: unset;
```
[the remark this answers is not in the document]
Comment (habemuscode) [reply views/scss/item-creator.scss:799]: We don't use Babel? I will try to found another way then

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: views/scss/item-creator.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsconan) [reply views/scss/item-creator.scss:799]: Babel is for the JS code. We use SASS for CSS. And I don't think there is a polyfill for this property.

[the remark this answers is not in the document]
Comment (habemuscode) [reply views/scss/item-creator.scss]: @jsconan Changed to `initial` and test that works

[the remark this answers is not in the document]
Comment (jsconan) [reply views/scss/item-creator.scss]: Checked, and bad luck. [initial is not supported](https://developer.mozilla.org/en-US/docs/Web/CSS/initial#Browser_compatibility) either. Did you check it under IE? On my testing env it is not working.

File: views/scss/item-creator.scss [hunk 2]
```diff
@@ -794,6 +794,12 @@ img.dragging.ui-draggable-dragging {
                     min-width: 100%;
                 }
 
+                .qti-include {
+                    img {
+                        min-width: initial;
```
Comment (jsconan) [inline views/scss/item-creator.scss:799]: Checked, and bad luck. [initial is not supported](https://developer.mozilla.org/en-US/docs/Web/CSS/initial#Browser_compatibility) either. Did you check it under IE? On my testing env it is not working.

**Review by jsconan — CHANGES_REQUESTED** [review]
Still not workinng under IE11

File: views/scss/item-creator.scss (hunk 2 above)
Comment (habemuscode) [reply views/scss/item-creator.scss]: Sorry I don't test again, VM open ralentize all. Bad, not check docs too.

I check with `inherit` and I push changes

imagen

**Review by AndreyNegrov — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by jsconan — APPROVED** [review]
Yeah! It works.
Testes under Firefox, Chrome, and IE11

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**